setup.py

```diff
@@ -14,19 +14,13 @@ def read(fname):
 
 
 install_requires = [
-    'mezzanine >= 3.1.10',
     'django-treebeard',
-    'django-filer>=0.9.6',
-    'South',
+    # pyScss fixed until https://github.com/fusionbox/django-pyscss/pull/26 is fixed.
+    'pyScss==1.2.1',
```

Comment:
The version pins are largely to get tests passing.  I'm open to removing any of the version changes or modifying them.  I will get this updated to your suggestion.